### ISSUE TITLE
Possibly update Compat to the most recent version

### ISSUE THREAD

**Issue by kafisatz** [JuliaCloud/AWS.jl#586]
Is it possible that you update Compat to a more recent version (>= 4.2)?

I a private package of mine I am using AWS with DataFrames and your  Compat requirement is holding back an update for DataFrames.

https://discourse.julialang.org/t/package-is-not-updating-but-why/88338/4

**Comment by mattBrzezinski** [comment]
There's an open [PR](https://github.com/JuliaCloud/AWS.jl/pull/585) made yesterday :)

Merging it in now!

**Comment by kafisatz** [comment]
thanks @mattBrzezinski 

For what it is worth I now have a compat & HTTP version restriction due to AWSS3.

```

(LLCalculationAgent) pkg> add DataFrames@1.4
   Resolving package versions...
ERROR: Unsatisfiable requirements detected for package HTTP [cd3eb016]:
 HTTP [cd3eb016] log:
 ├─possible versions are: 0.6.10-1.4.0 or uninstalled
 ├─restricted by compatibility requirements with AWS [fbe9abb3] to versions: 0.8.0-1.4.0
 │ └─AWS [fbe9abb3] log:
 │   ├─possible versions are: 1.0.0-1.78.0 or uninstalled
 │   ├─restricted to versions * by LLCalculationAgent [837fbea0], leaving only versions 1.0.0-1.78.0
 │   │ └─LLCalculationAgent [837fbea0] log:
 │   │   ├─possible versions are: 0.1.0 or uninstalled
 │   │   └─LLCalculationAgent [837fbea0] is fixed to version 0.1.0
 │   └─restricted by compatibility requirements with Compat [34da2185] to versions: 1.78.0 or uninstalled, leaving only versions: 1.78.0
 │     └─Compat [34da2185] log:
 │       ├─possible versions are: 1.0.0-4.3.0 or uninstalled
 │       └─restricted by compatibility requirements with DataFrames [a93c6f00] to versions: 4.2.0-4.3.0
 │         └─DataFrames [a93c6f00] log:
 │           ├─possible versions are: 0.11.7-1.4.0 or uninstalled
 │           ├─restricted to versions * by LLCalculationAgent [837fbea0], leaving only versions 0.11.7-1.4.0
 │           │ └─LLCalculationAgent [837fbea0] log: see above
 │           └─restricted to versions 1.4 by an explicit requirement, leaving only versions 1.4.0
 ├─restricted by compatibility requirements with AWS [fbe9abb3] to versions: 1.0.0-1.4.0
 │ └─AWS [fbe9abb3] log: see above
 └─restricted by compatibility requirements with AWSS3 [1c724243] to versions: 0.6.10-0.9.17 — no versions left
   └─AWSS3 [1c724243] log:
     ├─possible versions are: 0.4.0-0.10.1 or uninstalled
     ├─restricted to versions * by LLCalculationAgent [837fbea0], leaving only versions 0.4.0-0.10.1
     │ └─LLCalculationAgent [837fbea0] log: see above
     └─restricted by compatibility requirements with Compat [34da2185] to versions: 0.4.0-0.8.9 or uninstalled, leaving only versions: 0.4.0-0.8.9
       └─Compat [34da2185] log: see above

julia> 
```

**Comment by mattBrzezinski** [comment]
Ah I'll make a PR for this now! Ty for mentioning it
